src/utils/notifications.ts

```diff
@@ -32,10 +32,17 @@ import { doesRoomHaveUnreadMessages } from "../Unread";
 // MSC2867 is not yet spec at time of writing. We read from both stable
 // and unstable prefixes and accept the risk that the format may change,
 // since the stable prefix is not actually defined yet.
-// Assuming is passes FCP with no changes, we should update to start writing
+// Assuming it passes FCP with no changes, we should update to start writing
 // the flag to the stable prefix (or both) and then ultimately use only the
 // stable prefix.
```

Comment:
I am willing to lay money that this comment will get forgotten if/when the change to stable prefixes is made. It's just not very obvious here. I feel like it would have a better chance of getting noticed inside `get,setMarkedUnreadState` ?

Reply:
Yeah, that might be more noticeable.